### ISSUE TITLE
Highlighting specific lines adds blank lines when copying

### ISSUE THREAD

**Issue by squidfunk** [squidfunk/mkdocs-material#3360]
### Discussed in https://github.com/squidfunk/mkdocs-material/discussions/3359

<div type='discussions-op-text'>

<sup>Originally posted by **andrewetchen** December 17, 2021</sup>
Hi everyone,

Using the copy-to-clipboard on code blocks with highlighted lines, extra lines are being added when pasted.

Using the [highlighting specific lines](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#highlighting-specific-lines) _Result_ as an example. The copy-to-clipboard produces extra lines when pasted:

![image](https://user-images.githubusercontent.com/28404165/146613465-1d14905d-4925-474b-9211-5314013836a3.png)

Produces:

```
def bubble_sort(items):

    for i in range(len(items)):

        for j in range(len(items) - 1 - i):

            if items[j] > items[j + 1]:
                items[j], items[j + 1] = items[j + 1], items[j]
```

Current `markdown_extensions`

```yaml
markdown_extensions:
  - admonition
  - attr_list
  - meta
  - pymdownx.details
  - pymdownx.highlight:
      linenums: true
  - pymdownx.inlinehilite
  - pymdownx.magiclink:
      repo_url_shortener: true
  - pymdownx.tabbed:
      alternate_style: true
  - pymdownx.superfences
```

Adding the following CSS is a workaround to get around this but not ideal:

```css
.highlight .hll {
  display: inline;
}
```

---

Best regards,
Andrew</div>

**Comment by abhiTronix** [comment]
Yes, I'm facing this bug too.

**Comment by squidfunk** [comment]
Fixed in be0a36d1b.

**Comment by squidfunk** [comment]
Released as part of 8.1.3.

**Comment by andrewetchen** [comment]
Thanks for shipping this so quickly @squidfunk
